src/containers/SearchPage/MainPanel.js

```diff
@@ -64,8 +64,10 @@ class MainPanel extends Component {
       : {};
 
     const hasPaginationInfo = !!pagination && pagination.totalItems != null;
-    const totalItems = searchParamsAreInSync && hasPaginationInfo ? pagination.totalItems : 0;
-    const listingsAreLoaded = !searchInProgress && searchParamsAreInSync && hasPaginationInfo;
+    const listingsLength = listings ? listings.length : 0;
+    const totalItems =
+      searchParamsAreInSync && hasPaginationInfo ? pagination.totalItems : listingsLength;
```

Comment:
So, the decision was that `totalItems` is not returned in this case?
If that's true, you should write some comment about this exceptional API behaviour.

Reply:
Here's an example meta object in the response:

<img width="216" alt="Screenshot 2020-04-09 at 16 27 43" src="https://user-images.githubusercontent.com/53923/78900109-0b76ed00-7a7f-11ea-9ecd-c787b5374579.png">

Reply:
Use that `paginationUnsupported` info in `const hasPaginationInfo`
and add some comment there about the reason.